__tests__/purchase-client.test.ts

```diff
@@ -0,0 +1,49 @@
+import PurchaseClient from "../src/purchase-client";
+
+describe('PurchaseClient', () => {
+  describe('DevPurchaseClient', () => {
+    beforeEach(() => {
+      window.MEDKIT_PURCHASABLE_ITEMS = [{
+        sku: "UNITTESTSKU01",
+        displayName: "Unit Test SKU 01",
+        cost: {
+            amount: 1,
+            type: "test-cost",
+          },
+        },
+        {
+          sku: "UNITTESTSKU02",
+          displayName: "Unit Test SKU 02",
+          cost: {
+           amount: 2,
+           type: "test-cost",
+          }
+        }
+      ];
+    });
```

Comment:
I thought we discussed moving this to a constant outside of the functions?

Reply:
Yes...a while after I checked this in.

Reply:
So are you still working on this?